.ci/setup_env_ubuntu.sh

```diff
@@ -36,7 +34,7 @@ chronic sudo apt-get install -y --force-yes qemu-lite
 clear_release=$(curl -sL https://download.clearlinux.org/latest)
 cc_img_path="/usr/share/clear-containers"
 
-"./$cidir/install_clear_image.sh" ${clear_release} "${cc_img_path}"
+"${TESTS_CI_DIR}/install_clear_image.sh" ${clear_release} "${cc_img_path}"
```

Comment:
I'd rather we don't start adding CI specific env variables as dependencies of these scripts. My end goal with those is to be able to call them for developers as well in order to setup or update their environment.

We can fix this another way, this keeping `cidir`, but removing the leading `.` I believe.